components/search.jsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+import Searchbar_Dropdown from '@/app/results/search_dropdown';
```

Comment:
I think this name will need to be changed. Name in file is Search_Dropdown but it's importing Search"_bar_"_Dropdown

Reply:
I think because it's a default export it will work anyways and import it as an "as Searchbar_Dropdown" (might be wrong) but just keeping things consistent between files is a good idea.